### ISSUE TITLE
Ground Truth TF's slow after update

### ISSUE THREAD

**Issue by dan-riley** [osrf/subt#659]
Since updating to the latest Ignition and SubT versions (as detailed in the update page), enableGroundTruth is essentially broken.  We have tried several different robot models, and they all publish extremely slow TF's (around 0.5-1hz it seems) and there seems to be some other massive cpu or memory usage happening. When running with ground truth off everything runs as it did before.  I'm not sure which update caused the change, but we're seeing it on multiple machines.

**Comment by nkoenig** [comment]
@dan-riley , did @acschang answer solve your problem?

**Comment by dan-riley** [comment]
@nkoenig Yes, this seems to work.  We used 50hz.  I agree that it should probably be moved off of the static TF.

**Comment by peci1** [comment]
Solution proposed in #759.
